package.json

```diff
@@ -28,6 +28,12 @@
     "synchronizeColors": "npx ts-node tools/importers/SynchronizeColorsTokens.ts"
   },
   "devDependencies": {
+    "@fortawesome/fontawesome-svg-core": "6.1.1",
```

Comment:
after moving it to `devDependencies` there is no need to install these dependencies in `wnl-platform`